lightning/src/ln/channelmanager.rs

```diff
@@ -92,8 +92,8 @@ use core::ops::Deref;
 pub(super) enum PendingHTLCRouting {
 	Forward {
 		onion_packet: msgs::OnionPacket,
-		/// The SCID from the onion that we should forward to. This could be a "real" SCID, an
-		/// outbound SCID alias, or a phantom node SCID.
+		/// The SCID from the onion that we should forward to. This could be a real SCID or a fake one
+		/// generated using `get_fake_scid` from the scid_utils::fake_scid module.
```

Comment:
Possible nit:
Crate link the `crate::util::scid_utils::fake_scid` module.

Reply:
That doc link wouldn't be exposed anywhere since it's a private struct :)

Reply:
Ah my bad :)